### DISCUSSION TITLE
Is Chef still under Apache License 2.0?

### DISCUSSION THREAD

**Discussion by tisonkun** [chef/chef#13189] [Q&A] [answered]
I read the licensed year on README.md ends in 2020. And there's a EULA on the Chef website: https://www.chef.io/end-user-license-agreement.

Thus, I wonder whether this software is still open-source software and if I can use it for any purpose as described in [OSD](https://opensource.org/osd) 4 & 5.

**Comment by kmf** [comment]
<img width="428" alt="image" src="https://user-images.githubusercontent.com/78700/190666041-6f72b677-7a7f-4d7a-8c86-7993557d97a4.png">

https://github.com/chef/chef/blob/main/LICENSE

**Reply by tisonkun** [reply]
I see this license. But as mentioned in the question, it violates other license agreements provided by chef.io.

**Reply by Stromweld** [reply, marked as answer]
The code is apache 2.0 licensed. The build binaries that Chef creates falls under the end user license agreement. This is similar to RedHat code. The name Chef names/logos for all it's products are Trademarked. Any derivatives have to ensure all trademarks and copyrighted works are removed. Similar to how CentOS was a derivative of RedHat.

**Reply by tisonkun** [reply]
Thanks!
